<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Register</class>
 <widget class="QDialog" name="Register">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>751</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>600</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 246, 251);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="regICON">
   <property name="geometry">
    <rect>
     <x>310</x>
     <y>30</y>
     <width>131</width>
     <height>111</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;br/&gt;&lt;img src=&quot;:/newPrefix/signup.png&quot;/&gt;&lt;/p&gt;&lt;p&gt;&lt;br/&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>150</y>
     <width>751</width>
     <height>51</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>10</pointsize>
     <weight>50</weight>
     <italic>false</italic>
     <bold>false</bold>
     <underline>false</underline>
     <strikeout>false</strikeout>
     <kerning>false</kerning>
    </font>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 10pt &quot;Nirmala UI&quot;;</string>
   </property>
   <property name="text">
    <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#aa557f;&quot;&gt;Please fill in all required fields &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#ff557f;&quot;&gt;in english&lt;/span&gt;&lt;span style=&quot; color:#aa557f;&quot;&gt; and click on &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#aa557f;&quot;&gt;register &lt;/span&gt;&lt;span style=&quot; color:#aa557f;&quot;&gt;to create an account&lt;/span&gt;&lt;/p&gt;&lt;p align=&quot;center&quot;&gt;&lt;span style=&quot; color:#aa557f;&quot;&gt;then click on &lt;/span&gt;&lt;span style=&quot; font-weight:600; color:#aa557f;&quot;&gt;login&lt;/span&gt;&lt;span style=&quot; color:#aa557f;&quot;&gt; To go to the login page&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="userNameField">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>230</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 11pt &quot;Nirmala UI&quot;;
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="emailField">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>280</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 11pt &quot;Nirmala UI&quot;;
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwordField">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>380</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 11pt &quot;Nirmala UI&quot;;
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="confirmpasswordField">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>430</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 11pt &quot;Nirmala UI&quot;;
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="registrr">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>510</y>
     <width>131</width>
     <height>41</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Nirmala UI</family>
     <pointsize>9</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton:hover{
	background-color:white;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;

}

QPushButton{
background-color: rgb(255, 207, 233);
border-color: qradialgradient(spread:pad, cx:0.5, cy:0.5, radius:0.5, fx:0.5, fy:0.5, stop:0 rgba(0, 0, 0, 0), stop:0.52 rgba(0, 0, 0, 0), stop:0.565 rgba(82, 121, 76, 33), stop:0.65 rgba(159, 235, 148, 64), stop:0.721925 rgba(255, 238, 150, 129), stop:0.77 rgba(255, 128, 128, 204), stop:0.89 rgba(191, 128, 255, 64), stop:1 rgba(0, 0, 0, 0));
border-radius:30px;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;

}</string>
   </property>
   <property name="text">
    <string> register</string>
   </property>
  </widget>
  <widget class="QPushButton" name="gotologin">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>560</y>
     <width>181</width>
     <height>31</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>9</pointsize>
     <underline>true</underline>
    </font>
   </property>
   <property name="cursor">
    <cursorShape>PointingHandCursor</cursorShape>
   </property>
   <property name="styleSheet">
    <string notr="true">QPushButton{
border:0px;
	background-color: rgb(255, 246, 251);
}
QPushButton:hover{

	background-color:white;
	border-top-left-radius: 10px;
	border-bottom-left-radius: 10px;
border-top-right-radius: 10px;
	border-bottom-right-radius: 10px;

}
</string>
   </property>
   <property name="text">
    <string>Do you want to log in? click here </string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>480</x>
     <y>240</y>
     <width>101</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>username:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_7">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>290</y>
     <width>121</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>email:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>390</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>440</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>confirm password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>460</x>
     <y>300</y>
     <width>47</width>
     <height>13</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="MobileNumField">
   <property name="geometry">
    <rect>
     <x>180</x>
     <y>330</y>
     <width>271</width>
     <height>41</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(0, 0, 0, 0);
font: 11pt &quot;Nirmala UI&quot;;
gridline-color: rgb(255, 255, 255);
border-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_6">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>340</y>
     <width>81</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Arial Narrow</family>
     <pointsize>10</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>phone number:</string>
   </property>
  </widget>
  <widget class="QLabel" name="error">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>470</y>
     <width>291</width>
     <height>41</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label_9">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>751</width>
     <height>20</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 221, 246);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
 </widget>
 <resources>
  <include location="icon.qrc"/>
 </resources>
 <connections/>
</ui>
